### PR TITLE
feat: generate single back cover for multi ils export

### LINKED ISSUE #80: For multi ILS export, display one back cover

### PR DESCRIPTION
**Pull request by pyphilia**
closes #80

### CODE REVIEW

File: src/config.js [hunk 1]
```diff
@@ -72,6 +72,21 @@ export const MODE_INTERACTIVE_ONLINE = 'interactive-online';
...
+whether the whole or part of the material is concerned, specifically the rights of translation, reprinting, reuse of 
+illustrations, recitation, broadcasting, reproduction on microfilms or in any other physical way, and transmission or 
+information storage and retrieval, electronic adaptation, computer software, or by similar or dissimilar methodology 
+now known or hereafter developed. Exempted from this legal reservation are brief excerpts in connection with reviews 
+or scholarly analysis or material supplied specifically for the purpose of being entered and executed on a computer system, 
+for exclusive use by the purchaser of the work. Duplication of this publication or parts thereof is permitted only under the 
+provisions of the Copyright Law of the Publisher’s location, in its current version, and permission for use must always be obtained 
+from Springer. Permissions for use may be obtained through RightsLink at the Copyright Clearance Center. Violations are liable to 
+prosecution under the respective Copyright Law.${TEXT_SEPARATOR}
+Graasp (graasp.eu) is developed at the École Polytechnique Fédérale de Lausanne (epfl.ch)
```
Comment (juancarlosfarah) [inline src/config.js]: Add a `.` at the end.

File: src/config.js [hunk 2]
```diff
@@ -72,6 +72,21 @@ export const MODE_INTERACTIVE_ONLINE = 'interactive-online';
...
+Generated in Switzerland by Graasp${TEXT_SEPARATOR}
+This work is subject to copyright. All rights are reserved by the Publisher, 
+whether the whole or part of the material is concerned, specifically the rights of translation, reprinting, reuse of 
+illustrations, recitation, broadcasting, reproduction on microfilms or in any other physical way, and transmission or 
+information storage and retrieval, electronic adaptation, computer software, or by similar or dissimilar methodology 
+now known or hereafter developed. Exempted from this legal reservation are brief excerpts in connection with reviews 
+or scholarly analysis or material supplied specifically for the purpose of being entered and executed on a computer system, 
+for exclusive use by the purchaser of the work. Duplication of this publication or parts thereof is permitted only under the 
+provisions of the Copyright Law of the Publisher’s location, in its current version, and permission for use must always be obtained 
+from Springer. Permissions for use may be obtained through RightsLink at the Copyright Clearance Center. Violations are liable to
```
Comment (juancarlosfarah) [inline src/config.js]: Springer -> Graasp.

Change the permissions sentence to: "Permissions for use may be obtained through the support page on the Graasp platform."

File: src/services/export.js [hunk 3]
```diff
@@ -815,12 +821,12 @@ const saveEpub = async (page, mode, lang, username, sectionNumber = null) => {
     }
   }
 
-  const backCoverContent = 'generated by Graasp.eu';
-  // title is necessary to not write over cover chapter
-  const backCover = { title: 'Back Cover', data: backCoverContent };
+  // title is needed for epub xml page path
+  // @TODO add data to section's back cover
+  const backCover = { title: 'Back Cover', data: '' };
```
Comment (juancarlosfarah) [inline src/services/export.js]: @pyphilia, we do not want to add the `title` to the 'Back Cover'. It should only show the text you mentioned above.

File: src/services/export.js [hunk 4]
```diff
@@ -1153,9 +1160,20 @@ const convertSpaceToFile = async (id, body, headers) => {
           .map(param => param.coverPath)
           .filter(cover => cover != null);
 
+        // merge back covers content
+        const allBackCovers = {
+          title: 'Back Cover',
```
Comment (juancarlosfarah) [inline src/services/export.js]: Again, here @pyphilia, we should never show a `title` for the back covers.

Comment (juancarlosfarah) [reply src/services/export.js]: Remember that there is only **one** back cover.

**Comment by pyphilia** [comment]
I let the possibility to add some subspace content inside the back cover, I don't know what, but it is possible. Otherwise I can just let this static back cover for every export.

**Comment by juancarlosfarah** [comment]
@pyphilia, OK. I will go ahead and merge this review.

**Pull request merged**